safe-global/safe-wallet-web
# fix: upgrade `@web3-onboard/*` + remove patches

## What it solves

Resolves #2197

## How this PR fixes it

The `@web3-onboard/*` packages have been upgraded to their latest release which improved WC v2 support (allowing us to remove our patch):

- WC
- Ledger
- Taho

Note: our CSS hack to remove the "Where's my wallet?" info box from the modal could also be removed.

## How to test it

Connect to the Safe with WC v2 (via wallet/Safe), Ledger or Taho and observe execution/signing working. No "Where's my wallet?" info box should be present in the connection modal either.

## Checklist
